packages/wallet-sdk/src/sign/scw/utils/createSubAccountSigner.ts

```
@@ -0,0 +1,125 @@
+        }
+        case 'personal_sign': {
+          assertArrayPresence(args.params);
+          return account.signMessage({ message: args.params[0] } as {
+            message: SignableMessage;
+          });
+        }
+        case 'eth_signTypedData_v4': {
+          assertArrayPresence(args.params);
+          return account.signTypedData(args.params[1] as TypedDataDefinition);
+        }
+        case 'eth_signTypedData_v1':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)